scout/server/blueprints/variants/templates/variants/cancer-variants.html

```diff
@@ -43,7 +43,7 @@
       </div>
       <!--Expand filters form if filters were used in a previous POST request or the previous POST request form was not validated-->
       <div class="card-body panel-collapse collapse {{ 'show' if expand_search is sameas true }}" id="collapseFilters" >
-         {{ cancer_filters(form, institute, case) }}
+        <div class="card-body">{{ cancer_filters(form, institute, case) }}</div>
```

Comment:
Why a card body inside another card body?

Reply:
Ah, good catch - I just added it per template from the other variants. Without it, there are no margins for the buttons, but quite agree. I'll see if I can't fix it for all instead - perhaps with some explicit margins.

Reply:
Ok, it was a little messy, but the main reason it didn't look good was a couple of missing `</div>` from the string coordinates collapse. The browser kind of solved it based on the hint from an extra body class, but not without. Reverted the others as well.